### PR TITLE
[CHANGELOG] Fix media quality

### PR DESCRIPTION
**Pull request by oliviertassinari**
A polish on #9422, these media look very strange on my screen, enough for me to want to fix it 😁

Before: https://github.com/LukasTy/mui-x/blob/25c67a67f2b30fb6a79324eb0d3be30cf3a21563/CHANGELOG.md#690

After: https://github.com/mui/mui-x/blob/21493776c80dc8ebd87ca6756d6ab4105d665217/CHANGELOG.md#690

(I can't figure out how to remove the bottom border line)

It's more or less the same fix as in https://github.com/mui/mui-x/pull/9225

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,17 +9,17 @@ _Jun 22, 2023_
 
 We'd like to offer a big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:
 
-- 🎁 We release a new package `@mui/x-charts`.
-  This package aims at simplifying the integration of charts into your dashboards. 📊
+- 🎁 We released a new open-source package: `@mui/x-charts`. This package aims at simplifying the integration of charts into your dashboards. 📊
 
-  ![image](https://github.com/mui/mui-x/assets/45398769/613bae2f-537d-4ec3-8046-591d1a504a72)
+  <img width="512" alt="charts" src="https://github.com/mui/mui-x/assets/3165635/41201d3c-16a4-442d-a230-68356e6b433d">
```
Comment (LukasTy) [inline CHANGELOG.md:14]: Nice touch replacing the demo with a vertical charts layout. 👍

**Review by LukasTy — APPROVED** [review]
LGTM. 👍

**Comment by oliviertassinari** [comment]
@LukasTy I think that the takeaway is that when we add media content (images, videos, gifs), we should test how they render on a high resolution screen, so I imagine in your case, your MacBook screen.

Then, once we identify a media quality issue, if these are screenshots or screen recordings, they have to be done from high-resolution screens, it doesn't work from external screens with low resolutions.

**Comment by LukasTy** [comment]
Yes, that is a very correct point. I will keep it in mind. 👍 
As for this case, the media was provided by feature authors (@flaviendelangle and @alexfauquette), so, if you do not have a device with at least 2x screen ratio, ping someone else who can create the necessary media. 👍

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,17 +9,17 @@ _Jun 22, 2023_
...
-  It already contains [lines](https://mui.com/x/react-charts/lines/), [bars](https://mui.com/x/react-charts/bars/), and [scatter](https://mui.com/x/react-charts/scatter/) charts with basic customization features.
-  Please, visit the new [documentation](https://mui.com/x/react-charts/) to know what it does, and open issues to get the feature you need to be implemented.
+  It already contains [line](https://mui.com/x/react-charts/lines/), [bar](https://mui.com/x/react-charts/bars/), and [scatter](https://mui.com/x/react-charts/scatter/) charts, with basic customization features. Check out the [documentation](https://mui.com/x/react-charts/) to see what it can do, and open issues to get the feature you need implemented.
 
-- 🚀 Introduce the UTC and timezone support for pickers.
-  ![image](https://github.com/mui/mui-x/assets/3309670/f8b552b0-9ec2-417b-b769-95740308291d)
-  Please visit the [documentation](https://mui.com/x/react-date-pickers/timezone/) to learn how to use it.
+- 🚀 Introducing UTC and timezone support for pickers.
+
+  <img width="774" height="314" src="https://github.com/mui/mui-x/assets/3165635/ad95a404-ee67-4aff-b996-ad6cbb322348">
```
[the remark this answers is not in the document]
Comment (LukasTy) [reply CHANGELOG.md:20]: It gets squished. 🙈 
On the other hand, I would be interested to see, how many people are reading a package changelog on a mobile. 🙈 
If we wanted to make those responsive, a change would have to be made to every usage of a media item. 🤔

[the remark this answers is not in the document]
Comment (oliviertassinari) [reply CHANGELOG.md:20]: > What happens if users are viewing this on mobile?

@richbustos This important to check 👍 , not a nit-pick IMHO, nice finding.

The `width` and `height` attributes are required to be present to solve the layout shift when the page is loading, it's a recommendation by Chrome's dev team: https://web.dev/browser-level-image-lazy-loading/#images-should-include-dimension-attributes. We require the same for our blog: https://www.notion.so/mui-org/Content-tips-5a1686758f4f4267989b67a1c2c270da?pvs=4#2c953181fe534200ac688d414ca0bf85.

However, GitHub doesn't handle this correctly 🙃, they should have done: https://web.dev/optimize-cls/#images-without-dimensions,

Screenshot 2023-06-28 at 19 31 49

(the source of the web.dev)

Instead, they only set the `max-width: 100%`, so we can only add the `width` attribute.

I have fixed this with d76a6f3db9736cbd3003b6b284d3b3b0ea1f6627. Both suck but I think that it's still better to have images that look correct on mobile than no layout shift on desktop.

**Pull request merged**